Fix broken links and update publication metadata

daniel-montalvo opened the pull request:
<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/aria/pull/2102.html" title="Last updated on Jan 15, 2024, 11:20 AM UTC (4a2375d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/2102/8d09545...4a2375d.html" title="Last updated on Jan 15, 2024, 11:20 AM UTC (4a2375d)">Diff</a>

pkra commented on index.html:
```diff
@@ -267,8 +268,8 @@ <h1>Important Terms</h1>
-			  <p>Operating systems and other platforms provide a set of interfaces that expose information about <a class="termref">objects</a> and <a class="termref">events</a> to <a>assistive technologies</a>. Assistive technologies use these interfaces to get information about and interact with those <a class="termref">widgets</a>. Examples of accessibility APIs are <a href="https://docs.microsoft.com/ ...
-				  <a href="https://developer.apple.com/documentation/appkit/nsaccessibility">Mac <abbr title="OS Ten">OS X</abbr> Accessibility Protocol</a> [[AXAPI]], the <cite><a href="https://developer.gnome.org/atk/unstable/">Linux/Unix Accessibility Toolkit</a></cite> [[ATK]] and <cite><a href="https://developer.gnome.org/libatspi/stable/">Assistive Technology Service Provider Interface</a></cite> [[AT- ...
+			  <p>Operating systems and other platforms provide a set of interfaces that expose information about <a class="termref">objects</a> and <a class="termref">events</a> to <a>assistive technologies</a>. Assistive technologies use these interfaces to get information about and interact with those <a class="termref">widgets</a>. Examples of accessibility APIs are <a href="https://docs.microsoft.com/ ...
+				  <a href="https://developer.apple.com/documentation/appkit/nsaccessibility">Mac <abbr title="OS Ten">OS X</abbr> Accessibility Protocol</a> [[AXAPI]], the <cite><a href="https://gnome.pages.gitlab.gnome.org/atk/">Linux/Unix Accessibility Toolkit</a></cite> [[ATK]] and <cite><a href="https://developer-old.gnome.org/libatspi/stable/">Assistive Technology Service Provider Interface</a></cite> [ ...
```
Maybe replace `https://developer-old.gnome.org/libatspi/stable/` with `https://gnome.pages.gitlab.gnome.org/at-spi2-core/libatspi/` ?

cc @joanmarie

daniel-montalvo replied:
Thanks @pkra  That one slipped through. Will replace.

pkra commented on index.html:
```diff
@@ -505,7 +506,7 @@ <h3>Information for Authors</h3>
       </p>
 	    <p>Typically, the author will use host language <a>semantics</a> to put the widget in the <kbd>Tab</kbd> sequence (e.g., <code>tabindex="0"</code> in HTML) and <code>aria-activedescendant</code> to point to the ID of the currently active descendant. The author, not the user agent, is responsible for styling the currently active descendant to show it has keyboard focus. The author cannot use  ...
-	    <p>More information on managing focus can be found in the <a href="#keyboard" class="practices">Developing a Keyboard Interface</a> section of the <cite><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</cite>.</p>
+	    <p>More information on managing focus can be found in the <a href="practices/keyboard-interface" class="practices">Developing a Keyboard Interface</a> section of the <cite><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</cite>.</p>
```
This relative path seems problematic. It won't resolve on github.io or w3.org as far as I can tell.

daniel-montalvo replied:
Interested. I would have sworn It did resolve in https://w3.org/TR/2024/WD-wai-aria-1.3-20240118/

Then I'd say we can use the absolute path for safety. https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/

Given that the practices are no longer in TR space we may want to drop them from the spec URLs section.

pkra replied:
Oh, sorry I had missed that connection! Thanks for clarifying!

pkra reviewed (commented):
Mostly questions.

spectranaut commented on index.html:
```diff
@@ -116,10 +116,11 @@
+          "FPWD": "https://w3c.github.io/aria/",
+          "WD": "https://w3c.github.io/aria/",
+          "CR": "https://w3c.github.io/aria/",
+          "REC": "https://w3c.github.io/aria/",
```
Should this be all pointing to the editor's draft? shouldn't they mostly be pointing to 1.3 first public working draft?

pkra replied:
I basically meant to ask the same question but did so on the other side of the diff.

spectranaut reviewed (commented):
just one question

daniel-montalvo commented:
@spectranaut @pkra  I think I've addressed most of your comments. ARE you happy for me to merge this?

spectranaut commented:
> @spectranaut @pkra I think I've addressed most of your comments. ARE you happy for me to merge this?

Daniel, I still have one question, all the URLS in the `ariaSpecURLs` object point to the editors draft at github.io -- so a lot of the links within the preview go to the editors draft. I thought they should point back to the FFWD?

daniel-montalvo reviewed (commented):
Please @pkra  and @spectranaut see my comments. They have been pending all these days because I marked them as "review" comments instead of "single comments".

I changed the links that were pointing to the editor's draft.

The practices lihnk I think it's OK. Otherwise I am not really understanding what @pkra  's concern is with it.

daniel-montalvo commented:
> > @spectranaut @pkra I think I've addressed most of your comments. ARE you happy for me to merge this?
> 
> Daniel, I still have one question, all the URLS in the `ariaSpecURLs` object point to the editors draft at github.io -- so a lot of the links within the preview go to the editors draft. I thought they should point back to the FFWD?
> 
> For example, the "searchbox" link in this section of the preview: https://pr-preview.s3.amazonaws.com/w3c/aria/pull/2102.html#introroles
> 
> All the links that are not relative links go back to the editor's draft.

That's now changed.
